src/bug_report.rs

```diff
@@ -0,0 +1,198 @@
+use crate::utils::exec_cmd;
+use std::fs;
+use std::path::PathBuf;
+
+pub fn create() {
+    let os_info = os_info::get();
+
+    let environment = Environment {
+        os_type: os_info.os_type(),
+        os_version: os_info.version().to_owned(),
+        shell_info: get_shell_info(),
+        starship_config: get_starship_config(),
+    };
+
+    let link = make_github_issue_link(crate_version!(), environment);
+
+    if open::that(&link)
+        .map(|exit_status| exit_status.success())
+        .unwrap_or(false)
+    {
+        println!(
+            "I was unable to launch your browser. You'll have to copy this link instead:\n\n{}",
+            link
+        );
+    }
+}
+
+const UNKNOWN_SHELL: &str = "<unknown shell>";
+const UNKNOWN_VERSION: &str = "<unknown version>";
+const UNKNOWN_CONFIG: &str = "<unknown config>";
+
+struct Environment {
+    os_type: os_info::Type,
+    os_version: os_info::Version,
+    shell_info: ShellInfo,
+    starship_config: String,
+}
+
+fn make_github_issue_link(starship_version: &str, environment: Environment) -> String {
+    let title = urlencoding::encode("Bug Report:");
+
+    let body = urlencoding::encode(&format!("## Bug Report
+
+#### Current Behavior
+<!-- A clear and concise description of the behavior. -->
+
+#### Expected Behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+#### Additional context/Screenshots
+<!-- Add any other context about the problem here. If applicable, add screenshots to help explain. -->
+
+#### Environment
+- Starship version: {}
+- {}: {}
+- Operating system: {} {}
+
+#### Relevant Shell Configuration
+
+```bash
+{}
+```
+
+#### Starship Configuration
+
+```bash
```

Comment:
This definitely needs to be changed in the bug template, but we might as well fix it here too 😛 
```suggestion
```toml
```